Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1665,3 +1669,63 @@ extension BrowserViewController: IntroViewControllerDelegate {
         })
     }
 }
+
+extension BrowserViewController: ContextMenuHelperDelegate {
+    func contextMenuHelper(contextMenuHelper: ContextMenuHelper, didLongPressElements elements: ContextMenuHelper.Elements) {
+        let actionSheetController = UIAlertController(title: nil, message: nil, preferredStyle: UIAlertControllerStyle.ActionSheet)
+        var dialogTitle: String?
+
+        if let url = elements.link {
+            dialogTitle = url.absoluteString
+            let newTabTitle = NSLocalizedString("Open In New Tab", comment: "Context menu item for opening a link in a new tab")
```

Comment:
@st3fan @darrinhenein We had "Open In New Tab", which this PR put back. It sounds like @st3fan is talking about another context menu item that also opens the link in a tab, but doesn't switch to it. Do we want this? (Note that we don't have such an option on Android.)